src/main/java/org/jabref/gui/help/VersionWorker.java

```diff
@@ -61,21 +61,13 @@ private Optional<Version> getNewVersion() throws IOException {
     }
 
     public void checkForNewVersionAsync() {
-        if (!internalPreferences.isVersionCheckEnabled()) {
-            return;
-        }
-
```

Comment:
Does this change mean, jabref will now always check on startup, if a new version is present?
This can be very annoying, if you explicitly want to stay on an older version, if the newer one has a bug.

Reply:
This method will only execute when using Help -> Check for updates, so it shouldn't perform any checks.

The other method, `checkForNewVersionDelayed()` (Line 70), used to check for updates on startup, will only be called if it is enabled in the preferences.
https://github.com/JabRef/jabref/blob/7e4e7d3fba39ed6c1813a522c3fac3e454165c8b/src/main/java/org/jabref/gui/JabRefGUI.java#L111-L119